assets/js/boldgrid-inspirations.js

```diff
@@ -488,24 +631,39 @@ IMHWPB.InspirationsDesignFirst = function( $, configs ) {
 	 * @since 1.2.5
 	 */
 	this.onResize = function() {
-
 		/*
-		 * When the window is resized, wait 0.4 seconds and readjust the highlighted device preview
-		 * button.
+		 * Pointers don't always do well when the window is resized. Elements may shift around on the
+		 * page and the pointers become "broken". If the user does resize the page while a pointer is
+		 * shown, just hide the pointer.
 		 */
-		$( window ).resize( function() {
-			clearTimeout( $.data( this, 'resizeTimer' ) );
+		self.closePointers();
 
-			$.data(
-				this,
-				'resizeTimer',
-				setTimeout( function() {
-					self.highlightDeviceButton();
-				}, 400 )
-			);
-		} );
+
+		// Wait 0.4 seconds and readjust the highlighted device preview button.
```

Comment:
I can see that this is waiting 0.4 seconds and then making the re-adjustment.....but WHY do we wait 0.4 seconds before doing this?